code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1741,6 +1741,21 @@
 	reagent_data = list(/decl/reagent/nutriment = list("hot stew" = 3, "spices" = 1, "vegetables" = 1, "fish" = 2))
 	reagents_to_add = list(/decl/reagent/nutriment = 8, /decl/reagent/water = 5)
 
+/obj/item/reagent_containers/food/snacks/soup/maeuntang
+	name = "konyanger maeuntang"
```

Comment:
```suggestion
	name = "maeuntang"
```
Should not have Konyang in it's name. No other food has this, neither should Konyang dishes. Use the description instead.

Reply:
I was actually going off of the Morozian brudet, and this has a Konyanger twist because of the moss (not present on the irl Korean dish), but if you're concerned about it we can remove it.